libraries/Ethernet/src/EthernetClient.cpp

```diff
@@ -188,12 +188,12 @@ size_t EthernetClient::write(const uint8_t *buf, size_t size)
 	return size;
 }
 
-int EthernetClient::available()
+int EthernetClient::available(int timeout=0)
 {
```

Comment:
I agree, adding the optional argument won't break coding relying on the timeout. There's a `int timeout`. In our project this will allow us to run our mantainance tasks without relying on Linux specific signals and timers that are not part of Arduino spec 😄